meta/main.yml

```diff
@@ -23,12 +23,15 @@ galaxy_info:
       versions:
         - stretch
         - buster
+        # We do not yet have a Debian package for Bullseye.
+        # Presumably we could obtain one from the CDM folks.
```

Comment:
I still don't see anything past Buster on the [list of supported client platforms](https://docs.tanium.com/client/client/requirements.html#Client_host_system_requirements).